zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2564,7 +2564,51 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
-  /mapping-rules/{mappingKey}:
+  /mapping-rules/{id}:
```

```suggestion
  /mapping-rules/{mappingRuleId}:
```

🔧 Prefer qualified names of IDs & keys, for clarity.

(I didn't mark other changes associated with this name change)

---

As I know the entity was called mapping-rule at the start of the project and later in time the Entity has changed to Mapping, I'm going to update the url in api as well, but with the last PR of mapping api change like this one: [#29614](https://github.com/camunda/camunda/pull/29614)